GameFramework/GameFramework.vcxproj

```
@@ -129,26 +129,36 @@
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
+    <ClCompile Include="Algorithm\Algorithm.cpp" />
     <ClCompile Include="GameFramework\DirectXParam\DirectXDevice\DirectX9Device.cpp" />
     <ClCompile Include="GameFramework\DirectXParam\DirectXDevice\DirectXDevice.cpp" />
     <ClCompile Include="GameFramework\DirectXParam\DirectXGraphicDevice\DirectX9GraphicDevice.cpp" />
     <ClCompile Include="GameFramework\DirectXParam\DirectXGraphicDevice\DirectXGraphicDevice.cpp" />
     <ClCompile Include="GameFramework\DirectXParam\DirectXInputDevice\DirectX8InputDevice.cpp" />
     <ClCompile Include="GameFramework\DirectXParam\DirectXInputDevice\DirectXInputDevice.cpp" />
+    <ClCompile Include="GameFramework\GameFramework.cpp" />
     <ClCompile Include="GameFramework\GameFrameworkFactory\GameFrameworkFactory.cpp" />
     <ClCompile Include="RectSize\RectSize.cpp" />
+    <ClCompile Include="RotaionValue\Degree.cpp" />
+    <ClCompile Include="RotaionValue\Radian.cpp" />
+    <ClCompile Include="RotaionValue\RotationValue.cpp" />
```

自動テストのプロジェクトのビルドが通らず四苦八苦してたら…
```suggestion
    <ClCompile Include="RotationValue\RotationValue.cpp" />
```
フォルダ名のタイポ！！

---

フォルダ名修正しました